app/components/delivery-table.js

```diff
@@ -9,43 +10,47 @@ const statusFilterFunction = function(status, selected) {
   return status.capitalize() == selected;
 };
 
-export default Ember.Component.extend({
+export default Component.extend({
   transfers: null, // required property duke-ds-project-transfers models
   currentDukeDsUser: null, // required property current user's duke-ds-user model
   currentUser: null, // required property current user model
   tagName: 'div',
-  outgoingTransfers: Ember.computed('transfers.[]', 'currentDukeDsUser', function() {
+  outgoingTransfers: computed('transfers.[]', 'currentDukeDsUser', function() {
     const currentDukeDsUserId = this.get('currentDukeDsUser.id');
     return this.get('transfers').filterBy('fromUser.id', currentDukeDsUserId);
   }),
-  incomingTransfers: Ember.computed('transfers.[]', 'currentDukeDsUser', function() {
+  incomingTransfers: computed('transfers.[]', 'currentDukeDsUser', function() {
     const currentDukeDsUserId = this.get('currentDukeDsUser.id');
     return this.get('transfers').filter(function(transfer) {
       return transfer.get('toUsers').mapBy('id').includes(currentDukeDsUserId);
     });
   }),
-  outgoingColumns: [
-    { propertyName: "project.name", title: "Project Name", routeName: "deliveries.show"},
-    { propertyName: "toUsersNames", title: "To"},
-    { propertyName: "status",
-      title: "State",
-      component: 'delivery-state',
-      filterWithSelect: true,
-      predefinedFilterOptions: statusOptions,
-      filterFunction: statusFilterFunction
-    }
-  ],
-  incomingColumns: [
-    { propertyName: "project.name", title: "Project Name", routeName: "deliveries.show"},
-    { propertyName: "fromUser.fullName", title: "From"},
-    { propertyName: "status",
-      title: "State",
-      component: 'delivery-state',
-      filterWithSelect: true,
-      predefinedFilterOptions: statusOptions,
-      filterFunction: statusFilterFunction
-    }
-  ],
+  init() {
+    this._super(...arguments);
+
+    this.outgoingColumns = [
```

Comment:
This change was in response to the linter instead of a codemod. See
https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/avoid-leaking-state-in-ember-objects.md
They directly assign the list property in the "Good" code instead of using a set method.
You can see all linter items at https://github.com/ember-cli/eslint-plugin-ember